docs/reference_guide.md

```diff
@@ -509,9 +509,9 @@ Examples in situ:
 
 ### 10. map.increment()
 
-Syntax: ```map.increment(&key)```
+Syntax: ```map.increment(key)```
 
-Increments the key's value by one. Used for histograms.
+Increments the key's value by one. Used for histograms. This is actually translated to `lookup()` and `update()` calls.
```

Comment:
if we want the doc to clarify such details then it probably needs to be more accurate and say that it's update(&key,&zero,noexist); value=lookup(&key); value++. Otherwise "lookup and update" is confusing.